From 2bdb47f7e04f19c4477a2bf671c1c24cbe6c04f1 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:37:29 -0400
Subject: [misc] mei: txe: put pm callbacks under PM_SLEEP ifdef

Message-id: <1399041470-21330-57-git-send-email-prarit@redhat.com>
Patchwork-id: 79346
O-Subject: [RHEL7.1 PATCH BZ 1088953 56/77] mei: txe: put pm callbacks under PM_SLEEP ifdef
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit e0270addae23aef8401b1150cbca20b8fb373cd0
Author: Tomas Winkler <tomas.winkler@intel.com>
Date:   Tue Feb 18 14:31:09 2014 +0200

    mei: txe: put pm callbacks under PM_SLEEP ifdef

    PCI suspend resume callbacks should be defined
    under CONFIG_PM_SLEEP

    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Alexander Usyskin <alexander.usyskin@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/pci-txe.c b/drivers/misc/mei/pci-txe.c
index af4412c..a5bf39a 100644
--- a/drivers/misc/mei/pci-txe.c
+++ b/drivers/misc/mei/pci-txe.c
@@ -209,7 +209,7 @@ static void mei_txe_remove(struct pci_dev *pdev)
 }
 
 
-#ifdef CONFIG_PM
+#ifdef CONFIG_PM_SLEEP
 static int mei_txe_pci_suspend(struct device *device)
 {
  struct pci_dev *pdev = to_pci_dev(device);
@@ -273,7 +273,7 @@ static SIMPLE_DEV_PM_OPS(mei_txe_pm_ops,
 #define MEI_TXE_PM_OPS (&mei_txe_pm_ops)
 #else
 #define MEI_TXE_PM_OPS NULL
-#endif /* CONFIG_PM */
+#endif /* CONFIG_PM_SLEEP */
 /*
  *  PCI driver structure
  */
-- 
1.7.1